### DISCUSSION TITLE
Can I copy one TW to create a new version

### DISCUSSION THREAD

**Discussion by Boomcoach** [simonbaird/tiddlyhost#230] [Q&A] [answered]
I have used Classic TiddlyWikis to run RPG campaigns, but with the saving issues, I moved away from it.  Tiddlyhost is a perfect solution, as I can save easily and also reach it easily from multiple devices.

In the past I had created a basic wiki that I would use as a template for new ones, to save set up, as I normally use much of the same layout for games.  Is there an easy way to create a copy of an existing wiki on Tiddlyhost?

**Comment by simonbaird** [comment, marked as answer]
#198 is open to suggest adding a clone option in the actions menu.

Until that feature gets added, you can do something similar. Here's how it might be done:
* Create a site called "boomcoach-template" or something similar.
* Set it up with all the customizations and layouts you want to use in your template
* Now use the "Download" option from the action menu to download it
* Find that downloaded file so you know where it is
* Now create a new site for a particular campaign
* As soon as you create it, go back to the sites list, find the new site, and click "Upload" from the actions menu
* Select the downloaded template file (from step 5) and follow the prompts to upload it to the new site
* Repeat as needed for each campaign

Does that make sense? Obviously it's less convenient than a clone menu option, but I think it would achieve the same result.

If you need to tweak or improve the template, remember to download a fresh copy of it.

**Reply by Boomcoach** [reply]
That should work well!  Its not like I will be doing it daily!

**Reply by Boomcoach** [reply]
I prefer to use Classic TW (old dog, new tricks, etc.).  When I create a new space, with the intent of uploading my default TW into it, does it matter if I select the Classic TW from Advanced Options when I create the space?

**Reply by simonbaird** [reply]
No, it doesn't make any difference.
